doc/LSST_DESC_Notes/template_LSST_DESC_Note.rst

```diff
@@ -1,53 +1,103 @@
 ..
```

Comment:
I think to insert the DESC logo as part of the header, you could do something like:
`.. image:: ./_static/desc-logo.png
   :width: 100 px
   :scale: 25 %
   :align: left`